awx/main/models/receptor.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+
+from awx.main.models.base import BaseModel
+from awx.main.models.ha import Instance
+
+
+class ReceptorNode(BaseModel):
+    NODE_TYPE_CHOICES = [
+        ('control', "Control plane node"),
+        ('hybrid', "Controller and execution"),
+        ('execution', "Execution plane node"),
```

Comment:
The hop node doesn't say "execution plane node". To avoid definition problems I'm prefer to avoid this language and just refer to the fact that it's execution-only functionality wise.

Reply:
These choices (except the hop node type) were copied directly from the choices for the `Instance` model.  If it is a problem here it's a problem there too.